CONTRIBUTING.md

```diff
@@ -52,6 +52,8 @@ yarn start
 
 Your site is now running at [`http://localhost:8000`](http://localhost:8000)!
 
+The first time you build the site locally, it will take upwards of 20 minutes. This is normal, and will take signifigantly less time afterwards.
```

Comment:
You could just do "it often takes 20+ minutes". Also I think you can lose the comma from the second sentence.